tests/e2e/specs/drawer.cy.js

```diff
@@ -35,4 +38,14 @@ describe('Drawer component', () => {
       .get('#toggle-drawer')
       .should('be.visible')
   })
+  it('should drag to trigger resize', () => {
+    cy.get('.v-navigation-drawer').invoke('innerWidth').should('be.eq', 260)
+    cy.get('.v-navigation-drawer__border')
+      .trigger('mousedown', { which: 1 })
+      .trigger('mousemove', { clientX: 0, clientY: 500 })
+      .trigger('mouseup', { force: true })
+    cy.get('.v-navigation-drawer__border')
+      .invoke('innerWidth').should('be.lt', 460)
```

Comment:
I think this makes more sense
```suggestion
    cy.get('.v-navigation-drawer')
      .invoke('innerWidth').should('be.eq', 0)
```